scripts/00150-6lackHat.py

```diff
@@ -0,0 +1,9 @@
+def Hello(full_name, ID, email, language):
+    return "Hello World, this is {} with HNGi7 ID {} and email address {} using {} for stage 2 task".format(full_name, ID, email, language)
```

Comment:
Hi, this format doesn't align with the requirements of the team. The email should come after the word `task` and a space. For more information check [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)

Reply:
Hey my script is yet to be added

Reply:
You still haven't added the email after the word task. Look at other people's output [here](https://github.com/ifatoki/hng-wonderwoman-task-1/pull/253)